packages/react-core/src/components/ProgressStepper/ProgressStep.tsx

```diff
@@ -22,10 +22,8 @@ export interface ProgressStepProps
   description?: string;
   /** ID of the title of the progress step. */
   titleId?: string;
-  /** Popover for a progress step. */
-  popover?: React.ReactNode;
-  /** @hide Forwarded reference to title container */
-  innerRef?: React.Ref<any>;
+  /** Forwards the stepRef to rendered function.  Use this prop to add a popover to the step.*/
+  render?: ({ stepRef }: { stepRef: React.RefObject<any> }) => React.ReactNode;
```

Comment:
Genuine question: why does `stepRef` need to be in an object? I pulled this branch locally and have `render` taking `stepRef` directly, and I *think* everything's still working properly.

Reply:
I was getting a lint error when I was using the innerRef we had before. I removed it so I no longer need an object.